docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+
+Each SDK provides APIs and abstractions to enable robust cancellation handling in Workflows and Activities.
+
+## Child Workflows
+
+All SDKs support the ability to spawn Child Workflows.
+
+| Feature                                 | Go  | Java | TypeScript | Python | .Net | PHP |
+| --------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Child Workflow APIs](#child-workflows) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+A Child Workflow Execution is a Workflow Execution that is spawned from within another Workflow.
+
+Child Workflows in Temporal SDKs refer to Workflows that are started from within another Workflow.
+
+Each SDK provides APIs and abstractions to handle the complexity behind the scenes.
+
+## Continue-As-New
+
+| Feature                             | Go  | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Continue as New](#continue-as-new) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+Continue-As-New is the mechanism by which all relevant state is passed to a new Workflow Execution with a fresh Event History.
+
+## Delay Start Workflow
+
+| Feature                                        | Go  | Java | TypeScript | Python | .Net | PHP |
+| ---------------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Delay Start Workflow](#delay-start-workflow)) | Yes | Yes  | No         | No     | No   | No  |
+
+<!-- Track feature: https://github.com/temporalio/features/issues/338 -->
+
+A Delay Start Workflow is a Workflow that waits a specified duration before spawn the first Workflow.
+If the Workflow receives a Signal before the delay, the Workflow will be started immediately and the rest of the delay is ignored.
+
+A Start with Signal won't trigger the Workflow to start immediately and follow the delay.
+
+## Dynamic Entities
+
+Temporal supports Dynamic Workflows, Activities, Signals, and Queries.
+
+These are unnamed entities that are invoked if no other statically defined entity with the given name exists.
+
+| Feature                                   | Go | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------------- | -- | ---- | ---------- | ------ | ---- | --- |
+| [Dynamic Activities](#dynamic-activities) | No | Yes  | No         | Yes    | Yes  | No  |
+| [Dynamic Queries](#dynamic-queries)       | No | Yes  | No         | Yes    | Yes  | No  |
+| [Dynamic Signals](#dynamic-signals)       | No | Yes  | Yes        | Yes    | Yes  | No  |
+| [Dynamic Workflows](#dynamic-workflows)   | No | Yes  | Yes        | Yes    | Yes  | No  |
+
+Dynamic Entities should be used judiciously as a fallback mechanism rather than the primary approach.
+Overusing them can lead to maintainability and debugging issues down the line.
+
+## Enhanced stack trace
```

Comment:
This is a lower level that is basically unused by anyone today (though UI will use it at some point). This doesn't deserve such prominence. Why do we mention this as a feature but don't even mention stack traces as a feature?